### PR TITLE
Ignore

### PR DESCRIPTION
**Pull request by camilamacedo86**
Ignore

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -32,11 +32,17 @@ jobs:
...
+    - stage: generate
       # The golden_test.sh check if the the testdata is updated according to the current changes
       # To update the testdata use the Makefile targets `make generate-setup` then `make generate-testdata`
       script: ./golden_test.sh
 
+    - stage: OSX Supportability
+      os: osx
+      # it will runs the golden.sh script which allows we check its supportability for the SO
+      script: 
+      - make generate
```
Comment (Adirio) [inline .travis.yml]:
```suggestion
      script: make generate
```

File: .travis.yml [hunk 2]
```diff
@@ -32,11 +32,17 @@ jobs:
       before_script: curl -sSfL https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh | sh -s -- -b $(go env GOPATH)/bin v1.23.3
       script: ./scripts/verify.sh
 
-    - stage: golden
+    - stage: generate
       # The golden_test.sh check if the the testdata is updated according to the current changes
       # To update the testdata use the Makefile targets `make generate-setup` then `make generate-testdata`
       script: ./golden_test.sh
```
Comment (Adirio) [inline .travis.yml]:
```suggestion
      script: make generate
```

**Comment by camilamacedo86** [comment]
HI @Adirio, 

> From a conceptual viewpoint, I'm not sure if we really need to run this test in every OS. This test is chekcing that testdata was updated properly, doing that once should be enough. Right?

See that what we are trying to address here is:

> How can we ensure that users will be able to use the Kubebuilder in Linux and Osx?

Then, not that. the golden.sh will run the mainly used commands with a good combination to scaffold the projects. So, if we exec all these commands successfully in an osx and Linux we can ensure that the project still working for both.

File: .travis.yml (hunk 2 above)
Comment (camilamacedo86) [reply .travis.yml]: The `make generate` just update and not check if it was updated or not which is the goal of this job.

**Pull request closed without merging**